README.md

```diff
@@ -220,7 +220,7 @@ Python:
 instrumentation.opentelemetry.io/inject-python: "true"
 ```
 
-DotNet:
+.NET ([docs](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.5.0/docs/README.md)):
```

Comment:
could we pin this to the main branch? Pinning this to a tag will get quickly outdatted

Reply:
We can take the burden of making sure that it is up-to-date when we are bumping the version. Once this is merged I will update our .NET auto-instrumentation releasing docs https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/releasing.md?plain=1#L56

I think this would be easier for the users. In future I would like to change it to opentelemetry.io. See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1876 for reference

Reply:
> We can take the burden of making sure that it is up-to-date when we are bumping the version.

 please document this somewhere, otherwise this will get forgotten

Reply:
@pavolloffay PTAL https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1995

Reply:
Maybe, it would make more sense to create a new target in the Makefile to update automatically this kind of stuff

Reply:
> Maybe, it would make more sense to create a new target in the Makefile to update automatically this kind of stuff

Good idea, but for sure I will not do it as part of this PR.